ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/Data_transformer_and_postprocessing_configuration_doc.md

```diff
@@ -0,0 +1,725 @@
+# Data Transformer and Postprocessing Configuration Doc
```

Comment:
this seems to be repeated in both the docs. let's keep the into in the file named introduction only

Reply:
How about I completely moving the Transformers to the other doc?